db.js

```diff
@@ -2,11 +2,15 @@ const Student = require("./models/Student");
 
 module.exports = {
 	addStudent: async function(studentObj) {
```

Comment:
this works for adding students, but not editing students - you'll want to update the `editStudentById` function as well.